x-pack/plugins/cloud_defend/public/components/control_general_view_selector/index.tsx

```diff
@@ -274,28 +274,39 @@ export const ControlGeneralViewSelector = ({
     [errorMap, index, conditionsAdded, onChange, selector, selectors]
   );
 
+  const onAddStringArrayCondition = useCallback(
+    (condition: SelectorCondition) => {
+      const updatedSelector = { ...selector, [condition]: [] };
+      onChange(updatedSelector, index);
+    },
+    [selector, onChange, index]
+  );
```

Comment:
Maybe here we need an additional controller for handling formError separate from input error? for example `hasFormError`